### ISSUE TITLE
'argument "context" is missing' error in .procBismarkCytosineReport

### ISSUE THREAD

**Issue by cookkate** [al2na/methylKit#198]
Updating to the most recent version of methylKit produced this error in a Bismark cytosine report dataset that I've been able to load in a previous methylKit version without issues (I believe it was 1.12.?). My hunch is that it's related to the additional context filtering that was recently added.

Code and data to reproduce is below, as well as the full error I'm seeing.

```
processed_file_list = as.list(c("asdf.txt.gz"))
sample_names = as.list(c("asdf"))
treatment_vector = c(1)
genome = "asdf"
rdata_dir = "/"

methRaw = methRead(location = processed_file_list,
                sample.id   = sample_names,
                treatment   = treatment_vector,
                assembly    = genome,
                context     = "CpG",
                mincov      = 10,
                pipeline    = "bismarkCytosineReport",
                header      = FALSE,
                skip        = 0,
                sep         = "\t",
                dbtype      = "tabix",
                dbdir       = rdata_dir
                )
```

Output is:

```Received list of locations.
Uncompressing file.
Reading file.
Error in .procBismarkCytosineReport(data, mincov) : 
  argument "context" is missing, with no default
```
[asdf.txt.gz](https://github.com/al2na/methylKit/files/4587539/asdf.txt.gz)

**Comment by alexg9010** [comment]
Hi @cookkate ,

Thanks for the good reproducible example! I fixed the issue, should be available via Bioconductor soon via version 1.14.1. 

Best,
Alex

**Comment by cookkate** [comment]
And thank you for the quick fix!
